code/modules/mob/living/basic/bots/medbot/medbot.dm

```diff
@@ -109,6 +109,8 @@
 	var/datum/techweb/linked_techweb
 	///our tipper
 	var/datum/weakref/tipper
+	///original description
+	var/original_desc
```

Comment:
u dont need this var, u can do initial(desc) to get the original description whenever